docs/balloon.md

```diff
@@ -0,0 +1,74 @@
+# Balloon
+
+Cloud Hypervisor emulates a balloon device based on the VIRTIO specification.
+Its main purpose is to provide the host a way to reclaim memory by controlling
+the amount of memory visible to the guest. But it also provides some interesting
+features related to guest memory management.
+
+## Parameters
+
+`BalloonConfig` (known as `--balloon` from the CLI perspective) contains the
+list of parameters available for the balloon device.
+
+```rust
+struct BalloonConfig {
+    pub size: u64,
+    pub deflate_on_oom: bool,
+    pub free_page_reporting: bool,
+}
+```
+
+```
+--balloon <balloon>	Balloon parameters "size=<balloon_size>,deflate_on_oom=on|off,free_page_reporting=on|off"
+```
+
+### `size`
+
+Size of the balloon device. It is substracted from the VM's total size. For

```

Comment:
nit: Typo.

Reply:
Where's the typo?

Reply:
> Where's the typo?

"substracted" ?

Reply:
Well I learned something today, I thought `subtract` was written `substract` :rofl: